### DISCUSSION TITLE
目前note13机型是没办法解锁了吗

### DISCUSSION THREAD

**Discussion by lwb-2021** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#216] [General]
红米note13机型目前全系列绑定帐号服务失效，显示系统升级维护已经很久了，有传言称就算升级到澎湃OS成功答题也没办法解锁

**Comment by xbn2002** [comment]
经测试表明，外版 note13 系列在升级 HyperOS 后可以使用外区资格成功绑定
国行暂不清楚升级 HyperOS 后是否可以使用国内社区资格绑定。
